tests/integration_tests/nodes_communication.py

```diff
@@ -42,6 +42,10 @@ def get_celery_create_view_signature(celery_app):
     return celery_app.signature("mipengine.node.tasks.views.create_view")
 
 
+def get_celery_create_pathology_view_signature(celery_app):
```

Comment:
all these signatures could be in a dict, it would be much more packed and clean..